README.md

```diff
@@ -1 +1,49 @@
-# Phil-s-Portfolio
\ No newline at end of file
+# Phil-s-Portfolio
+
+> Portfolio: Mobile Version
+
+**This is a portfolio website created by Phil using the template provided in Figma.**
+
+## Built With
+
+- HTML
+- CSS
+
+## Materials required for validation
+
+- follow this link :
+ [HTML and CSS Linters link]( https://github.com/microverseinc/linters-config/tree/master/html-css)
+
+## Screenshot
+
+![readme](../Phil-s-Portfolio/images/phil.png)
+
```

Comment:
- [ ] `required`
- [ ] Kindly fix this broken screenshot for the readme to look even better :v: